solarstations.csv

```diff
@@ -748,3 +748,7 @@ Bratislava,,,Slovakia,48.16983,17.07145,185,2016-2018,,Solargis,,,Upon request,G
 M'Bour,,,Senegal,14.394387,-16.958739,10,2007-2020,,University of Lille,,,,G;B;D
 Dakar,,,Senegal,14.7017,-17.4256,10,2020-,,University of Lille,,,,G;B;D
 Villeneuve d'Ascq,,,France,50.611208,3.140441,63,2008-,,University of Lille,,,,G;B;D
+Ashalim,,,Israel,30.9830,34.7080,305,2016-,,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
+Jesrusalem Givat Ram,,,Israel,31.7704,35.1973,770,1990-,,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
+Besor Farm,,,Israel,31.2716,34.38941,110,2001-,,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
+Bet Dagan,,,Israel,32.0073,34.8138,31,1991-,,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
```

Comment:
```suggestion
Ashalim,,,Israel,30.9830,34.7080,305,2016-,IMS,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
Jesrusalem Givat Ram,,,Israel,31.7704,35.1973,770,1990-,IMS,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
Besor Farm,,,Israel,31.2716,34.38941,110,2001-,IMS,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
Bet Dagan,,,Israel,32.0073,34.8138,31,1991-,IMS,IMS,Unknown if DHI is measured with shadowband,https://ims.gov.il/en/stations,Freely,G;B;D
```
I would call both the owner and network for IMS. Otherwise, excellent job and thanks for taking care of this!